pkg/controller/storagecluster/storagecluster.go

```diff
@@ -63,11 +63,12 @@ import (
 
 const (
 	// ControllerName is the name of the controller
-	ControllerName                      = "storagecluster-controller"
+	ControllerName = "storagecluster-controller"
+	// ComponentName is the component name of the storage cluster
+	ComponentName                       = "portworx"
```

Comment:
We don't have any portworx specific logic in the controller. All of it is in the driver/portworx package. To keep this portworx agnostic, can we call the annotation locator `pod/storage`?  Customers are used to storagenodes anyway, so storage pod would be on the same lines. Wdyt?
Sorry for pointing this out so late.
cc: @haibinxie

Reply:
sounds good.